set.mm

```diff
@@ -133784,6 +133854,12 @@ subset of a (possibly extended) finite sequence of integers.  (Contributed
       PVBUMTUN $.
   $}
 
+  $( An integer range between 1 and 2 is a pair.  (Contributed by AV,
+     11-Jan-2023.) $)
+  fz12pr $p |- ( 1 ... 2 ) = { 1 , 2 } $=
+    ( c1 c2 cfz co caddc cpr df-2 oveq2i cz wcel wceq ax-mp 1p1e2 preq2i 3eqtri
+    1z fzpr ) ABCDAAAEDZCDZARFZABFBRACGHAIJSTKPAQLRBAMNO $.
```

Comment:
For this kind of pairs (and in the applications below), I would use 0 and 1 instead of 1 and 2.

Reply:
This theorem is used for the case ``( EEhil ` 2 ) = ( RR^ ` ( 1 ... 2 ) ) = ( RR^ ` { 1 , 2 })``. In contrast, ``( EEhil ` 0 ) = ( RR^ ` (/) )`` should be a space of dimension 0, consisting of one (neutral) element only, at least according to the comment for ~df-ehl ("The case ` n = 0 ` corresponds to a space of dimension 0, that is, limited to a neutral element.") This is, however, not proven anywhere, is it? Therefore, the range cannot be changed without any major impacts related to the definition of ~df-ehl.

Reply:
I added a proof ~ehl0 of this statement in the comment of ~df-ehl, which should make clear why the range starts with 1 instead of 0.